<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImagedemoClass</class>
 <widget class="QMainWindow" name="ImagedemoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>945</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Imagedemo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>601</width>
      <height>521</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>945</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionOpen_Image"/>
    <addaction name="actionRestore_Image"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>点处理</string>
    </property>
    <addaction name="actionColor_gray_scale"/>
    <addaction name="actionImage_inversion"/>
    <addaction name="separator"/>
    <addaction name="actionVertical_mirror"/>
    <addaction name="actionHorizontal_mirror"/>
    <addaction name="action_180_degree_rotation"/>
    <addaction name="action30_degree_rotation"/>
    <addaction name="separator"/>
    <addaction name="actionAffine_transformation"/>
    <addaction name="actionPerspective_transformation"/>
    <addaction name="separator"/>
    <addaction name="actionBrightness_transformation"/>
    <addaction name="actionGrayscale_image_histogram"/>
    <addaction name="actionHistogram_equalization"/>
   </widget>
   <widget class="QMenu" name="menuLiyu">
    <property name="title">
     <string>邻域处理</string>
    </property>
    <addaction name="actionNeighborhood_average"/>
    <addaction name="actionGauss_Filter"/>
    <addaction name="actionMedian_filter"/>
    <addaction name="separator"/>
    <addaction name="actionSobel_Boundary_detection"/>
    <addaction name="actionLaplace_Boundary_detection"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>二值化</string>
    </property>
    <addaction name="actionSelect_the_threshold"/>
    <addaction name="actionAdaptive_threshold"/>
    <addaction name="actionBased_on_global_threshold"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>二值化处理</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>形态学处理</string>
    </property>
    <addaction name="actionCorrosion"/>
    <addaction name="actionExpand"/>
    <addaction name="actionOpen_operation"/>
    <addaction name="actionClosed_operation"/>
    <addaction name="actionMorphological_gradient"/>
    <addaction name="separator"/>
    <addaction name="actionTop_hat_transformation"/>
    <addaction name="actionTop_hat_transformation_2"/>
   </widget>
   <widget class="QMenu" name="menu_6">
    <property name="title">
     <string>彩色图像处理</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_7">
    <property name="title">
     <string>综合处理</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_8">
    <property name="title">
     <string>动态检测</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_9">
    <property name="title">
     <string>视频采集播放</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_10">
    <property name="title">
     <string>图像采集及其它</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_11">
    <property name="title">
     <string>关于</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menuLiyu"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
   <addaction name="menu_6"/>
   <addaction name="menu_7"/>
   <addaction name="menu_8"/>
   <addaction name="menu_9"/>
   <addaction name="menu_10"/>
   <addaction name="menu_11"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQwdq">
   <property name="text">
    <string>qwdq</string>
   </property>
  </action>
  <action name="actionOpen_Image">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Open Image</string>
   </property>
  </action>
  <action name="actionRestore_Image">
   <property name="text">
    <string>Restore Image</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>   Exit</string>
   </property>
  </action>
  <action name="actionColor_gray_scale">
   <property name="text">
    <string>Color gray scale</string>
   </property>
  </action>
  <action name="actionImage_inversion">
   <property name="text">
    <string>Image inversion
</string>
   </property>
  </action>
  <action name="actionVertical_mirror">
   <property name="text">
    <string>Vertical mirror</string>
   </property>
  </action>
  <action name="actionHorizontal_mirror">
   <property name="text">
    <string>Horizontal mirror</string>
   </property>
  </action>
  <action name="action_180_degree_rotation">
   <property name="text">
    <string>180 degree rotation</string>
   </property>
  </action>
  <action name="action30_degree_rotation">
   <property name="text">
    <string>30 degree rotation</string>
   </property>
  </action>
  <action name="actionAffine_transformation">
   <property name="text">
    <string>Affine transformation</string>
   </property>
  </action>
  <action name="actionPerspective_transformation">
   <property name="text">
    <string>Perspective transformation</string>
   </property>
  </action>
  <action name="actionBrightness_transformation">
   <property name="text">
    <string>Brightness transformation</string>
   </property>
  </action>
  <action name="actionGrayscale_image_histogram">
   <property name="text">
    <string>Grayscale image histogram</string>
   </property>
  </action>
  <action name="actionHistogram_equalization">
   <property name="text">
    <string>Histogram equalization</string>
   </property>
  </action>
  <action name="actionNeighborhood_average">
   <property name="text">
    <string>Neighborhood average</string>
   </property>
  </action>
  <action name="actionGauss_Filter">
   <property name="text">
    <string>Gauss Filter</string>
   </property>
  </action>
  <action name="actionMedian_filter">
   <property name="text">
    <string>Median filter</string>
   </property>
  </action>
  <action name="actionSobel_Boundary_detection">
   <property name="text">
    <string>Sobel  Boundary detection</string>
   </property>
  </action>
  <action name="actionLaplace_Boundary_detection">
   <property name="text">
    <string>Laplace Boundary detection</string>
   </property>
  </action>
  <action name="actionSelect_the_threshold">
   <property name="text">
    <string>Select the threshold</string>
   </property>
  </action>
  <action name="actionAdaptive_threshold">
   <property name="text">
    <string>Adaptive threshold</string>
   </property>
  </action>
  <action name="actionBased_on_global_threshold">
   <property name="text">
    <string>Otsu</string>
   </property>
  </action>
  <action name="actionCorrosion">
   <property name="text">
    <string>corrosion</string>
   </property>
  </action>
  <action name="actionExpand">
   <property name="text">
    <string>expand</string>
   </property>
  </action>
  <action name="actionOpen_operation">
   <property name="text">
    <string>Open operation</string>
   </property>
  </action>
  <action name="actionClosed_operation">
   <property name="text">
    <string>Closed operation</string>
   </property>
  </action>
  <action name="actionMorphological_gradient">
   <property name="text">
    <string>Morphological gradient</string>
   </property>
  </action>
  <action name="actionTop_hat_transformation">
   <property name="text">
    <string>Top hat transformation</string>
   </property>
  </action>
  <action name="actionTop_hat_transformation_2">
   <property name="text">
    <string>Trough detection</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="imagedemo.qrc"/>
 </resources>
 <connections/>
</ui>
